src/components/Game.tsx

```diff
@@ -0,0 +1,65 @@
+import { type } from "@testing-library/user-event/dist/type";
+import { useMemo, useState } from "react";
+import Board from "./Board";
+
+function Game() {
+  const resetbox: Array<string> = Array(9).fill("");
+  const [xIsNext, setxIsNext] = useState<boolean>(true);
```

Comment:
For `useState`, you can explicitly declare the type like this or omit it. If you omit it, it will take the type from initial value.